docs/markdown/snippets/static-analyzer.md

```diff
@@ -0,0 +1,5 @@
+## Added support for static analyzer
+
+User can use gcc(>= 10.0.0) and clang(>= 9.0.1) compiler, static analyzer feature with
```

Comment:
The static analyzer feature is available as `-Db_static_analyzer=true/false` on `gcc >= 10.0.0` and `clang >= 9.0.1`. In the event that `b_static_analyzer` is `true`, but the compiler doesn't support the flag, the value of `b_static_analyzer` is ignored.